packages/core/src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -62,6 +74,26 @@ function ProductDetails({ context: staleProduct }: Props) {
     },
   } = data
 
+  const [skuPropertyName] = useMemo(() => {
+    return Object.keys(skuVariants.activeVariations)
+  }, [skuVariants.activeVariations])
```

Comment:
Does this still work for products with more than 1 varying property? 🤔

Reply:
Actually, I don't think you can compute this here. You don't have enough information yet.

Reply:
We did this to solve the problem that it was hardcoded for using `Color` as the dominant one, so at least it would work with another facet name. We didn't test the behavior with more than one because there're no examples on our current store. 🫠

Reply:
But I don't think you can do that at this point just yet. The links need to be calculated inside the `Selectors` component, because each SKU Selector has to know its `propertyName`, and then use it to create these links